test/scala/com/github/johnynek/bazel_deps/ModelGenerators.scala

```diff
@@ -66,7 +66,7 @@ object ModelGenerators {
     heads <- Gen.option(Gen.listOf(Gen.identifier))
     cache <- Gen.option(Gen.oneOf(ResolverCache.Local, ResolverCache.BazelOutputBase))
     prefix <- Gen.option(Gen.identifier.map(NamePrefix(_)))
-  } yield Options(vcp, dir, langs, res, trans, heads, cache, prefix)
+  } yield Options(vcp, dir, langs, res, trans, heads, cache, prefix, None)
```

Comment:
can we update the tests so they do some testing?

Always putting None here is leaving the writing and parsing unchecked. I'd like to have a proper generator here.

Reply:
I wasn't 100% sure how to add a test and validate against the final BUILD output. Looks like a lot of the testing is autogenerated?

I did replace None with a random set of licenses though